### DISCUSSION TITLE
AppCentre for React Native publish error

### DISCUSSION THREAD

**Discussion by nithinv-born** [codemagic-ci-cd/codemagic-docs#1671] [Q&A]
I'm getting this error while publishing React Native Android app to AppCentre

`failed with exception "Missing required option -f / --file"`

My build id 6464779d8218e4e8dbb12e7b

**Comment by himesh-cm** [comment]
Hi @nithinv-born, what is the value of `$ARTIFACT_PATH` you have set?
From the error, it seems it isn't able to find the file.

**Reply by nithinv-born** [reply]
Hi @himesh-cm $ARTIFACT_PATH I have tried different values 
`android/app/build/outputs/**/*.aab`
`android/app/build/outputs/release/app-release.aab`
`android/app/release/app-release.aab`
`build/outputs/release/app-release.aab`
In yaml file, it is
`artifacts:
            - android/app/build/outputs/**/*.aab`
Sometimes it shows this error too 
`failed with exception "ENOENT: no such file or directory,`
 Please give me which path I have to use

**Comment by nithinv-born** [comment]
Issue resolved with below path set as an environment variable and artifacts

`android/app/build/outputs/apk/release/app-release.apk`
